### PR TITLE
List vaccine

### PR DESCRIPTION
**Pull request by maruf12**
Changes:
- added list vaccine request
- added filter vaccine request

Related:
https://airtable.com/appMJE57FaKQcEUwA/tbl6no9dhW3sDRJhb/viwittlbwGyGABPNQ?blocks=bipBVobkbQcj0ovZ8

Evidence
title: Added list and filter vaccine request
project: Pikobar logistik
participants: @maruf12, @naufalihsank, @bangunbagustapa, @Ibwedagama, @maulanayuseph, @yoslie, @adzharamrullah

### CODE REVIEW

File: src/components/requestVaccine/completenessDetail.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<template>
+  <v-dialog
+    v-model="show"
+    width="40%"
+    :persistent="true"
+  >
+    <v-card>
+      <div class="col-sm-12">
+        <v-card-text class="headerSection headerMargin">
```
Comment (naufalihsank) [inline src/components/requestVaccine/completenessDetail.vue:9]: camel case class

**Review by bangunbagustapa — APPROVED** [review]

File: src/components/requestVaccine/searchFilter.vue [hunk 2]
```diff
@@ -0,0 +1,201 @@
+<template>
+  <div>
+    <v-card-text>
+      <v-row class="margin-top-bot-min-20-list-pengajuan-logistik">
```
Comment (Ibwedagama) [inline src/components/requestVaccine/searchFilter.vue:4]: minor:

hello, @maruf12 I just wondering why this class is so long? is this class correctly written?

**Review by Ibwedagama — APPROVED** [review]

File: src/components/requestVaccine/searchFilter.vue (hunk 2 above)
Comment (naufalihsank) [reply src/components/requestVaccine/searchFilter.vue:4]: it's the legacy class from the first developer of this project 😂

**Review by maulanayuseph — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: src/components/requestVaccine/completenessDetail.vue (hunk 1 above)
Comment (maruf12) [reply src/components/requestVaccine/completenessDetail.vue]: @naufalihsank thanks for review guys, please check for update

File: src/components/requestVaccine/searchFilter.vue (hunk 2 above)
Comment (maruf12) [reply src/components/requestVaccine/searchFilter.vue]: @Ibwedagama @naufalihsank thanks for review guys, please check for update

File: src/components/requestVaccine/dataTable.vue [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    </v-data-table>
+  </div>
+</template>
+<script>
+export default {
+  name: 'DataTable',
+  props: {
+    items: {
+      type: Array,
+      default: null
```
[the remark this answers is not in the document]
Comment (maruf12) [reply src/components/requestVaccine/dataTable.vue]: @yoslie thanks for review guys, please check for update

**Review by yoslie — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**